en/docs/fragments/save-sp-in-portal.md

```diff
@@ -0,0 +1,3 @@
+After all the configurations click **Update**.
```

Comment:
This need not be a fragment...we can just have it as a step on the relevant page itself right?